jobs-builder/jobs/dependencies.yaml

```diff
@@ -60,18 +60,23 @@
       - shell:
           !j2: |
             #!/bin/bash
-            set -e
-            set -x
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            [ -n "$BASH_VERSION" ] && set -o errtrace
+            {% raw %}
+            [ -n "${DEBUG:-}" ] && set -o xtrace
+            {% endraw %}
 
             export GOPATH="${WORKSPACE}/go"
             export GOROOT="/usr/local/go"
+            export PATH="$PATH:$GOPATH/bin:$GOROOT/bin"
 
-            sudo apt install -y golang
             source ci/lib.sh
             export branch="{{branch}}"
             export target_branch="$branch"
             {% if flavor == "experimental" -%}
-            export experimental_kernel=true
+            export build_type="experimental"
```

Comment:
Hello @wainersm . I want to add a new arch-specific build_type, maybe it can be called "arch-experimental", to distinguish with this comment one.  So, what about exporting this new build_type here?
I'm trying to do this [here](https://github.com/kata-containers/kata-containers/pull/3281/commits/92ee825b2aa106e959ed8cb13c0dfe512f91b992), but have not settled down. 
@jodh-intel

Reply:
@jodh-intel @jongwu

it's been a moving target :) What do you prefer:
a) merge as is; send a follow up when things are settle down
b) keep changing and merge after a settlement

I'm in favor of a)

Reply:
@jodh-intel @wainersm -, I'm fine for a). let's do the following after it settles down.

Reply:
@wainersm, @jongwu - (a) wfm too! ;)

Reply:
great! done!